### PR TITLE
Connectors first try

### PR DESCRIPTION
**Pull request by andrewxhill**
- My understanding of @balupton's first use of connectors was more around complete SDKs that package these use-cases, but
- prior to having those, it seems helpful to just group the components of those connectors in sections of the docs, this will help us measure there interest and scope their potential without a lot of coding.
- added an Images & Media, it was instructive in that I can already see enhancements we'd need to make this really clean.
- I think it's a pretty good starting point though!

Also moved the clients docs down into the Develop section.

Not sure this is any more helpful for preview than just reading the markdown, but here you go.

# Connectors Introduction 

Screen Shot 2019-05-14 at 3 36 32 PM

# Images & Media

Screen Shot 2019-05-14 at 3 36 38 PM
Screen Shot 2019-05-14 at 3 36 43 PM
Screen Shot 2019-05-14 at 3 36 47 PM
Screen Shot 2019-05-14 at 3 36 50 PM

### CODE REVIEW

File: docs/connectors/images/index.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    }
+  }
+}
+```
+
+Unlike the CameraRoll schema, you can see that images are resampled into a few different variants but the original, full-resolution file is not maintained. You may also notice that the small format of a Media thread is the same format as the thumbnail in the CameraRoll thread. This can be helpful if you add a photo to multiple threads, as there is the opportunity to deduplicate this content.
+
+### Connect to existing photo threads
+
+In many cases, it is just as easy for your app to connect to a user's existing photo threads. Connecting to an existing stream can give your app the ability to provide value across many more photos or simplify photo selection, editing, or sharing workflows. Requesting permission to a user's existing Media or CameraRoll threads is straightforward in your app and is expected to be released to production in the first half of July. Subscribe to this Issue to track our progress on the feature.
```
Comment (balupton) [inline docs/connectors/images/index.md]: Missing link

File: docs/connectors/images/index.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+## Get Started
+
+Handling images for your users falls generally into two use-cases. First, images need to be store in full-resolution form, typically useful for backup & recovery or editing apps. Second, images need to be formatted for efficient distribution to other users or on the web. Here, we'll walk through setting up each use-case in your app.
+
+## Images & media threads
+
+Threads help a user grant access to their photos & media to new applications, share individual photos to new users, or sync all their data to trusted peers like their desktop or store it encrypted on the cafes.
+
+We covered threads in depth in the Tour of Textile and in the Threads overview pages. We recommend reviewing those two documents if you haven't already.
```
Comment (balupton) [inline docs/connectors/images/index.md]: > Tour of Textile

> Threads overview

Should be links.

File: docs/connectors/images/index.md (hunk 1 above)
Comment (andrewxhill) [reply docs/connectors/images/index.md]: nice. using this one https://github.com/textileio/go-textile/issues/694

**Comment by andrewxhill** [comment]
sweet thanks!

**Comment by andrewxhill** [comment]
going to move forward with 2/3 of your proposals. I got the RN todos all up to date. the ios and android are a big task and it'll be good to have the new urls to clients etc in place if we launch this tour blog post

**Pull request merged**